setup.py

```diff
@@ -75,9 +75,13 @@ def run(self):
                 f"-DLLVM_TARGETS_TO_BUILD=host",
                 f"-DMLIR_ENABLE_BINDINGS_PYTHON=ON",
                 f"-DLLVM_ENABLE_PROJECTS=mlir",
-                f"-DLLVM_EXTERNAL_PROJECTS=torch-mlir;torch-mlir-dialects",
+                f"-DTORCH_MLIR_ENABLE_MHLO=ON",
+                f"-DLLVM_EXTERNAL_PROJECTS=torch-mlir;torch-mlir-dialects;mlir_hlo",
                 f"-DLLVM_EXTERNAL_TORCH_MLIR_SOURCE_DIR={src_dir}",
                 f"-DLLVM_EXTERNAL_TORCH_MLIR_DIALECTS_SOURCE_DIR={src_dir}/externals/llvm-external-projects/torch-mlir-dialects",
+                f"-DLLVM_EXTERNAL_MLIR_HLO_SOURCE_DIR={src_dir}/externals/mlir-hlo",
+                # FIXME: set MLIR_PDLL_TABLEGEN_EXE since mlir-hlo doesn't
+                f"-DMLIR_PDLL_TABLEGEN_EXE=mlir-pdll",
```

Comment:
where do you get this tool binary from ? Is it built as part of our build ? Is it a host tool ? We cross-compile for arm64 and have to specify the host version like we do for llvm-tblgen.

Reply:
I think for now you want to point to the mlir-pdll that we built in the CMAKE_BINARY_DIR (or equivalent) or you will likely pick up the system one ? 

I will take care of the host / cross-compile whenever we land that in the future. So just have to confirm this mlir-pdll is the version we just built.

Reply:
I would like to use the mlir-pdll that we built in the CMAKE_BINARY_DIR. I raised a PR to MHLO to remove this config: https://github.com/tensorflow/mlir-hlo/pull/41